geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/RedisStatsIntegrationTest.java

```diff
@@ -473,28 +473,32 @@ public void NetworkKiloBytesReadOverLastSecond_shouldReturnCorrectData() {
 
   @Test
   public void clientsStat_withConnectAndClose_isCorrect() {
+    GeodeAwaitility.await().atMost(Duration.ofSeconds(2))
```

Comment:
What is this change for? Seems unrelated to the PR.

Reply:
> While testing, I noticed that there was a problem with the stat for `connected clients`. Occasionally this test and the next one would fail. My fix seems to have addressed the issue.

Reply:
I doubt you ran it enough times since it's clearly not the correct fix. The real problem is that when the connection is `closed` the stat which counts the connections is updated asynchronously. This means that if you read the stat immediately after a `close` it may not be correct. You need to wait for the stat to 'settle' before using it.

Reply:
> I'm not sure I follow you. I've run this fix a bunch of times on my laptop and it always passes.